src/hooks/api/useFetchExercises.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from 'react-query';
+import { getExercises } from '@/api';
+import { Exercise } from '@/types';
+
+const useFetchExercises = () => {
+  const { isLoading, error, isError, data } = useQuery<Exercise[], Error>(
+    'exercise',
+    () => getExercises(),
```

Comment:
```suggestion
    getExercises,
```
굳이 익명 함수로 하나 감싸지 않고 이렇게 표현도 가능할것 같습니다!

Reply:
위 경우에는 가능하지만 원래는 getExercise에 인수를 넘겨서 사용하는거라 저렇게 해논것 같네요. 반영해놓겠습니다~

Reply:
디코에서 얘기했듯이 타입 추론 이슈가 있기에 리팩토링 과정에서 더 살펴보고 수정하는걸로 가시죠~